### DISCUSSION TITLE
colors error after arranging electrode groups

### DISCUSSION THREAD

**Discussion by dlinsen1** [petersenpeter/CellExplorer#104] [Q&A] [answered]
I receive this error after arranging electrode groups, saving, and trying to reopen.

![image](https://github.com/petersenpeter/CellExplorer/assets/16261590/2060f82e-2469-4031-9ceb-4532b9749063)

**Comment by petersenpeter** [comment, marked as answer]
This is likely caused by a misconfiguration in the session metadata struct. You can share your session struct and I can take a look, or find documentation [here](https://cellexplorer.org/datastructure/data-structure-and-format/#session-metadata) and an example struct [here](https://buzsakilab.nyumc.org/datasets/PetersenP/CellExplorerExampleData/MS22/Peter_MS22_180629_110319_concat/Peter_MS22_180629_110319_concat.session.mat).
